docs/endpoints/posts.md

```diff
@@ -14,6 +14,8 @@ Only admins and post owners will have `tags`, `source`, `remote_addr` (which wil
 
 Anonymous requests will only return accepted posts. Logged-in users can see accepted posts & any of their own pending or rejected posts. Staff can see anything!
 
+If a post is anonymous, only `northstar_id` and `location` will be returned for the post owner.
```

Comment:
Can we re-word this to "the `northstar_id` field will only be returned for the post owner"? The current wording makes me think if a post is anonymous, I'd only get the `northstar_id` and `location` fields in the response.

Reply:
ah yes much clearer!